src/server.js

```
@@ -1,51 +1,56 @@
-// Personal Finance Simulator - A tool to simulate financial scenarios and make informed decisions
-const express = require('express');
-const bodyParser = require('body-parser');
-const cors = require('cors');
-const morgan = require('morgan');
-const path = require('path');
-const fs = require('fs');
-
-// Initialize express app
-const app = express();
-const PORT = process.env.PORT || 3002;
-
-// Middleware
-app.use(cors());
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(morgan('dev'));
+// Personal Finance Simulator
+var express = require('express');
+var bodyParser = require('body-parser');
+var cors = require('cors');
+var morgan = require('morgan');
+var path = require('path');
+var fs = require('fs');
+
+// Global variables
+var app = express();
+var PORT = 3002; // Hardcoded port number
+
+// Add middleware
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(morgan('dev'))
 
 // Serve static files from the public directory
 app.use(express.static(path.join(__dirname, '../public')));
 
-// Ensure data directory exists
-const dataDir = path.join(__dirname, '../data');
-if (!fs.existsSync(dataDir)) {
-  fs.mkdirSync(dataDir);
-}
-
-// Initialize scenarios file if it doesn't exist
-const scenariosFile = path.join(dataDir, 'scenarios.json');
-if (!fs.existsSync(scenariosFile)) {
-  fs.writeFileSync(scenariosFile, JSON.stringify([], null, 2));
-}
-
-// API Routes
-const scenariosRouter = require('./routes/scenarios');
+// Data setup
+var dataDir = path.join(__dirname, '../data');
+try {
+  if (!fs.existsSync(dataDir)) {
+    fs.mkdirSync(dataDir);
+  }
+} catch(e) { /* ignore errors */ }
+
+// Initialize file
+var scenariosFile = path.join(dataDir, 'scenarios.json');
```

Using empty catch blocks with `/* ignore errors */` is not a good practice. It's better to log errors or handle them appropriately, especially for file system operations that could fail for various reasons.